app/src/main/java/agency/five/tmdb/MainActivity.kt

```diff
@@ -1,43 +1,30 @@
     package agency.five.tmdb
 
+
```

Comment:
In the Help tab in Android Studio look for "Reformat code" action - use that regularly to reformat your code in files. Your indentations, spaces, empty lines, new lines are all over the place.